blazor/datagrid/how-to/using-different-edit-mode-for-add-edit-action.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Using different edit mode for add and edit action in Blazor DataGrid | Syncfusion
```

Comment:
Title: Using different edit mode for add and edit action in the Blazor DataGrid | Syncfusion